src/test/ui/associated-types/hr-associated-type-bound-2.stderr

```diff
@@ -1,12 +1,10 @@
-error[E0599]: no method named `f` found for type `u32` in the current scope
+error[E0599]: the method `f` exists for type `u32`, but the following trait bounds were not satisfied:
+`<u32 as X<'b>>::U: Clone`
+which is required by `u32: X`
   --> $DIR/hr-associated-type-bound-2.rs:19:10
    |
 LL |     1u32.f("abc");
    |          ^ method not found in `u32`
-   |
-   = note: the method `f` exists but the following trait bounds were not satisfied:
-           `<u32 as X<'b>>::U: Clone`
-           which is required by `u32: X`
 
```

Comment:
What do you think if we instead output:

```
error[E0599]: the method `f` exists for type `u32`, but its trait bounds were not satisfied
  --> $DIR/hr-associated-type-bound-2.rs:19:10
   |
LL |     1u32.f("abc");
   |          ^ method not found in `u32` due to unsatisfied trait bounds
   |
   = note: the following trait bounds were not satisfied:
           `<u32 as X<'b>>::U: Clone`
           which is required by `u32: X`
```

Reply:
That looks good, except for the usage of 'not found' in "method not found in `u32` due to unsatisfied trait bounds". I think it should say something like "method cannot be called on `u32` due to unsatisfied trait bounds".